scipy/signal/_filter_design.py

```diff
@@ -788,11 +788,12 @@ def freqz_sos(sos, worN=512, whole=False, fs=2*pi):
 
     See Also
     --------
-    freqz, sosfilt
+    freqz, sosfilt, sosfreqz
 
     Notes
     -----
-    .. versionadded:: 0.19.0
+    .. versionadded:: 1.15.0
+    This function used to be called sosfreqz in older versions (>=0.19.0)
```

Comment:
The missing empty line here cause a failure on the documentation build in CI. 

```suggestion
    This function used to be called ``sosfreqz`` in older versions (≥ 0.19.0)
    
    .. versionadded:: 1.15.0
    
```
As a matter of (my) personal taste, it might be nicer to switch the lines and highlight `sosfreqz`.

PS: As noted in this [checklist], you can build the documentation [locally] to verify that everything is correct.

[checklist]: https://docs.scipy.org/doc/scipy/dev/contributor/development_workflow.html#checklist-before-submitting-a-pr
[locally]: https://docs.scipy.org/doc/scipy/dev/contributor/rendering_documentation.html

Reply:
Thanks for the suggestions!